File: Flask/microservices/distance-service/website/point.py
from static.utils import attributes


class Point():
    def __init__(self, place_id_1: int, place_id_2: int, latitude_1: float, latitude_2: float, longitude_1: float, longitude_2: float) -> None:
        self.place_id_1 = place_id_1
        self.place_id_2 = place_id_2
        self.latitude_1 = latitude_1
        self.latitude_2 = latitude_2
        self.longitude_1 = longitude_1
        self.longitude_2 = longitude_2

    def as_dict(self) -> dict:
        return attributes(self)

File: Dotnet/microservices/distance-service/Models/Point.cs
namespace DistanceService.Models
{
    public class Point
    {
        public int PlaceId1 { get; set; }
        public int PlaceId2 { get; set; }
        public double Latitude1 { get; set; }
        public double Longitude1 { get; set; }
        public double Latitude2 { get; set; }
        public double Longitude2 { get; set; }

        public Point()
        {

        }

        public Point(int id1, int id2, double lat1, double lon1, double lat2, double lon2)
            => (PlaceId1, PlaceId2, Latitude1, Longitude1, Latitude2, Longitude2) = (id1, id2, lat1, lon1, lat2, lon2);
    }
}
